Settings.qml

```diff
@@ -45,7 +45,8 @@ Item {
             }
             TextField {
                 text: updaterSettings.commandLine
-                onAccepted: updaterSettings.commandLine = this.text
+                onEditingFinished: updaterSettings.commandLine = this.text
+                validator: RegExpValidator { regExp: /.*%command%.*/ }
```

Comment:
This isn't super good UX. Maybe do the validation in onEditingFinished?

Reply:
Well, I don't want to spend a lot of time on this now. I can remove the validator if you think that's better.

Reply:
I think we can trivially make this better by assuming that if `%command%` isn't there, then the arguments that are saved should be appendeded to `%command%`

Reply:
Good idea, done.